README.md

```diff
@@ -13,10 +13,11 @@ Planned support:
 * [x] Multi-user x86_64 Linux with systemd init
 * [ ] Multi-user aarch64 Linux with systemd init
 * [x] Multi-user x86_64 MacOS
-    + Note: Uninstall and encrypted volume support are incomplete
+    + Note: user deletion is still buggy
```

Comment:
Should we maybe document that it's currently un-implemented (due to potential issues with secure token stuff), not just that it's "buggy"?